assets/base.css

```diff
@@ -2072,3 +2073,20 @@ details-disclosure > details {
     transform: translateY(0);
   }
 }
+
+.overflow-hidden-mobile,
+.overflow-hidden-tablet {
+  overflow-y: hidden;
+}
+
```

Comment:
Since the original class doesn't target x or y (just `overflow`). What do you think in terms of naming? 🤔 

Maybe `overflow-vertical-hidden`? Or what do you think?

.overflow-hidden {
  overflow: hidden;
}

Reply:
Maybe it makes sense to just target both in reality.  Changed it to `overflow: hidden`.